src/transactions/slice.ts

```diff
@@ -57,12 +57,14 @@ interface State {
   standbyTransactions: StandbyTransaction[]
   transactionsByNetworkId: TransactionsByNetworkId
   feedFirstPage: TokenTransaction[]
+  hasNewUnknownCompletedTransactions: boolean
```

Comment:
```suggestion
  hasNewCompletedTransactions: boolean
```
maybe we don't need the "unknown" part? i was confused about whether "unknown" is a type of classification

Reply:
@kathaypacific it's purely for readability purposes as there's already an existing `hasNewlyCompletedTransactions` which is for the transactions that have just turned from pending to completed, therefore "newly completed". 
So we'll have both:
```
hasNewCompletedTransactions
hasNewlyCompletedTransactions
```

Considering that these names are almost identical, I thought that it might introduce a bit of a confusion as it's gonna be really easy to mistype or let IDE autocomplete the wrong one. And in order to not introduce a few extra lines of comments, I figured it might be enough of a distinction to have:
- `hasNewlyCompletedTransactions` - for transaction that have just turned from pending to completed
- `hasNewUnknownCompletedTransactions` - for transaction that are new and completed, but unknown as we did not know about them at all (they were not in our pending/failed/completed list previously)

Reply:
oh sorry i didn't realise we had something like `hasNewlyCompletedTransactions`! any chance we could just do the vibration when that flag is set, maybe in [this hook](https://github.com/valora-inc/wallet/blob/0f2064968f3bb052f8ccf86902ac839239178b22/src/transactions/feed/TransactionFeedV2.tsx#L262-L282)? as an aside, i haven't tested and it might not be a problem, but i have found in the past that having non primitive state variables results in unnecessary rerenders unless we do some special identity management (i haven't thought about this deeply so i don't have suggestions, but it did occur to me that the vibrations might trigger more than expected due to this if we were to put them here....but it's easy enough to have a different `useEffect` with carefully chosen dependency array variables to overcome this)

Reply:
@kathaypacific don't think it would be suitable in that particular hook as it's only responsibility is to update the state and side-effects shouldn't be called from within state setters. But there is a way to do it with a separate hook, that will use useEffects to track when the vibration for a specific set of transactions occurred.

I'll work on different version of haptic feedback and see if I can come up with a better solution.

Reply:
okay sure, sounds good. i do wonder how we should think about haptic feedback though, i can see why you say it is a side effect, but at the same time it is a synchronous action that doesn't trigger any meaningful _effects_ as such...

Reply:
@kathaypacific It is indeed a synchronous action, but from the perspective of redux state management and our implementation of it - it has to be _reactionary_ to the changes to the data. Unless, there's a way for me to catch a specific action is saga, _before_ the data gets synced up with the persisted feed (as described in the comment below).